### PR TITLE
[MERGE] lunch: Improve perf + general UX (back2basics)

### PR DESCRIPTION
**Pull request by tivisse**
TaskID: 2586326

### CODE REVIEW

File: addons/lunch/models/lunch_product_category.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+class LunchProductCategory(models.Model):
+    """ Category of the product such as pizza, sandwich, pasta, chinese, burger... """
+    _name = 'lunch.product.category'
+    _inherit = 'image.mixin'
+    _description = 'Lunch Product Category'
+
+    @api.model
+    def _default_image(self):
+        image_path = get_module_resource('lunch', 'static/img', 'lunch.png')
+        return base64.b64encode(open(image_path, 'rb').read())
```
Comment (mart-e) [inline addons/lunch/models/lunch_product_category.py:20]:
```suggestion
        image = file_open('lunch/static/img/lunch.png', 'rb').read()
        return base64.b64encode(image)
```

File: addons/lunch/security/ir.model.access.csv [hunk 2]
```diff
@@ -17,4 +17,3 @@ lunch_topping_user,access_lunch_topping,model_lunch_topping,group_lunch_user,1,0
 lunch_topping_manager,access_lunch_topping,model_lunch_topping,group_lunch_manager,1,1,1,1
 lunch_supplier_user,"Lunch Supplier User Rights",model_lunch_supplier,group_lunch_user,1,0,0,0
 lunch_supplier_manager,"Lunch Supplier Manager Rights",model_lunch_supplier,group_lunch_manager,1,1,1,1
-lunch_product_report_user,access_lunch_product_report,model_lunch_product_report,group_lunch_user,1,1,0,0
```
Comment (mart-e) [inline addons/lunch/security/ir.model.access.csv:20]: :seal_of_approuval:

**Review by mart-e — COMMENTED** [review]
@robodoo override=ci/security
Reason: the image path is static and safe content (but can be rewritten to avoid to use `open`)

**Comment by tivisse** [comment]
@robodoo r+ merge

File: addons/web/static/src/webclient/commands/command_palette.scss [hunk 3]
```diff
@@ -60,10 +60,6 @@
       line-height: 1.8em;
       height: 2.8em;
 
-      &.focused {
-        background: #00a09d26;
-      }
-
```
Comment (brboi) [inline addons/web/static/src/webclient/commands/command_palette.scss:66]: Isn't this a mistake ?

**Pull request merged**